backend/app.py

```
@@ -0,0 +1,203 @@
+        return {"success": False, "response": f"Chat service error: {str(e)}"}
+    
+    return {"success": False, "response": "Chat service unavailable"}
+
+@app.get("/")
+async def root():
+    """Health check endpoint"""
+    return {"message": "DIY Visual Finder API is running"}
+
+if __name__ == "__main__":
+    import uvicorn
+    uvicorn.run(app, host="0.0.0.0", port=8000)
```

Ruff (0.16.0)

[error] Possible binding to all interfaces

(S104)